frontend/src/components/InputText.js

```diff
@@ -1,8 +1,63 @@
 import React, {Component} from 'react';
+import InputTextItems from './InputTextItems'
+import './InputText.css'
 
 class InputText extends Component {
+    constructor(props) {
+        super(props);
+       
+        this.state = {
+          items: []
+        };
+       
+        this.addItem = this.addItem.bind(this);
+        this.deleteItem = this.deleteItem.bind(this);
+
+      }
+
+    addItem(e) {
```

Comment:
nit: replace e with a descriptive name.

Reply:
react documentations use e for synthetic events, should I follow documentation or change the variable